edx_proctoring/backends/software_secure.py

```diff
@@ -349,6 +353,19 @@ def _split_fullname(self, full_name):
 
         return (first_name, last_name)
 
+    def _create_zendesk_ticket(self, review, serialized_exam_object, serialized_attempt_obj):
+        """
+        Creates a Zendesk ticket for review with status Suspicious
```

Comment:
Looks like you moved the statuses to notify on to the instance variable `create_zendesk_for_status`. Can you update this comment?